specification/recoveryservicesbackup/resource-manager/readme.md

```diff
@@ -37,7 +37,7 @@ license-header: MICROSOFT_MIT
 ```
 
 ```yaml $(package-passivestamp)
-tag: package-2024-10
+tag: package-passivestamp-2023-01-15
```

Comment:
swagger avacado check is failing due to this change. You may need to specify only one 'tag' option.

Also this tag value is incorrect

Reply:
@venu-sivanadham we always had multiple tags, and this is like this in every release. We are just pointing to the right tag for passive stamp.

Reply:
what is the purpose these passive/active sections ?

Avacado tool doesn't seems to accept different package versions.

Reply:
@venu-sivanadham this is to cater an old model of our service where we had a different set of APIs with different version 2023-01-15 to support a Disaster recovery scenario for Cross region restore.  It's just there for information purpose from quite long if someone wants to generate clients for passive stamp APIs for CRR

Reply:
If you want specific api version to be used for passive stamp then you can add those commentary/details under the respective version's ### Tag:  section (At line 158) as normal text.

Since defining tag multiple times under ## Basic Information section is unexpected, the Swagger Avocado check continue to complain every time you modify this line.

This is not a blocking comment but I would recommend to address this issue.